commands/utils/packagehandlers/packagehandlers_test.go

```
@@ -1,27 +1,151 @@
+		},
+		shouldFix: false,
+	}
+	cleanup := createTempFolderAndChDir(t, testDataDir, coreutils.Maven)
+	defer cleanup()
+	shouldFix, err := mvn.UpdateImpactedPackage(test.impactedPackage, test.fixVersionInfo)
+	assert.NoError(t, err)
+	assert.Equal(t, test.shouldFix, shouldFix)
+}
+
+// Until implemented, test we are not attempting to fix indirect dependencies in python
+func TestPythonPackageHandler_UpdateImpactedPackage(t *testing.T) {
+	testDataDir := getTestDataDir(t)
+	testcases := []pythonIndirectPackages{
+		{
+			indirectPackageFixTest: indirectPackageFixTest{impactedPackage: "urllib3",
+				fixVersionInfo: &utils.FixVersionInfo{
+					FixVersion:       "1.25.9",
+					PackageType:      "pip",
+					DirectDependency: false}, shouldFix: false},
+			requirementsPath:     "requirements.txt",
+			pythonPackageManager: "pip",
+		}, {
+			indirectPackageFixTest: indirectPackageFixTest{impactedPackage: "urllib3",
+				fixVersionInfo: &utils.FixVersionInfo{
+					FixVersion:       "1.25.9",
+					PackageType:      "poetry",
+					DirectDependency: false}, shouldFix: false},
+			requirementsPath:     "pyproejct.toml",
+			pythonPackageManager: "poetry",
+		}, {
+			indirectPackageFixTest: indirectPackageFixTest{impactedPackage: "urllib3",
+				fixVersionInfo: &utils.FixVersionInfo{
+					FixVersion:       "1.25.9",
+					PackageType:      "pipenv",
+					DirectDependency: false}, shouldFix: false},
+			requirementsPath:     "Pipfile",
+			pythonPackageManager: "pipenv",
+		},
+	}
+	for _, test := range testcases {
+		t.Run(test.pythonPackageManager.ToString(), func(t *testing.T) {
+			packageHandler := GetCompatiblePackageHandler(test.fixVersionInfo, &utils.ScanDetails{
+				Project: utils.Project{PipRequirementsFile: test.requirementsPath}}, nil)
+			cleanup := createTempFolderAndChDir(t, testDataDir, test.fixVersionInfo.PackageType)
+			defer cleanup()
+			shouldFix, _ := packageHandler.UpdateImpactedPackage(test.impactedPackage, test.fixVersionInfo)
```

Let's assert and not ignore the error returns by this function.